tests/constants/session.constants.ts

```diff
@@ -21,10 +21,5 @@ export const NO_OPEN_PHASES_SESSION_DATA: SessionData = {
 
 export const MULTIPLE_OPEN_PHASES_SESSION_DATA: SessionData = {
   ...BUG_REPORTING_PHASE_SESSION_DATA,
-  openPhases: [
-    Phase.phaseBugReporting,
-    Phase.phaseTeamResponse,
-    Phase.phaseTesterResponse,
-    Phase.phaseModeration
-  ]
+  openPhases: [Phase.phaseBugReporting, Phase.phaseTeamResponse, Phase.phaseTesterResponse, Phase.phaseModeration]
```

Comment:
Let's remove these formatting changes, since they are unrelated to this PR.
Seems that we accidentally missed some test code when adopting prettier --> can open a separate PR for this.